pkg/runner/healthcheck_common.go

```diff
@@ -0,0 +1,283 @@
+package runner
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"os"
+	"os/exec"
+
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/client"
+	"github.com/docker/go-connections/nat"
+	"github.com/docker/go-units"
+	"github.com/ipfs/testground/pkg/api"
+	"github.com/ipfs/testground/pkg/docker"
+	"github.com/ipfs/testground/pkg/rpc"
```

Comment:
Import order, in groups:

- stdlib
- testground
- third parties

https://github.com/golang/go/wiki/CodeReviewComments#imports